### PR TITLE
VIH-8380 staff member judge joins hearing from consultation room

### PR DESCRIPTION
**Pull request by robertbennett1998**
### JIRA link ###
[VIH-8380](https://tools.hmcts.net/jira/browse/VIH-8380)

### Change description ###
- Fixed join hearing button being shown to QL participants.
- Removed pointless test cases.

**Does this PR introduce a breaking change?**
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.ts [hunk 1]
```diff
@@ -58,7 +58,9 @@ export class PrivateConsultationRoomControlsComponent extends HearingControlsBas
 
     get canJoinHearingFromConsultation(): boolean {
         return (
-            this.conferenceStatus?.newStatus === ConferenceStatus.InSession && this.participant?.status === ParticipantStatus.InConsultation
+            this.conferenceStatus?.newStatus === ConferenceStatus.InSession &&
+            this.participant?.status === ParticipantStatus.InConsultation &&
+            this.isHost
```
Comment (omio-ma) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.ts:63]: should quick link participant be able to called into a hearing in session, I cant remember whether that was a requirement? This logic wouldn't impact that would it?

Comment (robertbennett1998) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.ts:63]: I aren't sure but the button wont be visible as it is only  shown when they are available. So it shouldn't impact anything.

**Review by ZJianWang — APPROVED** [review]
LGTM

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.ts (hunk 1 above)
Comment (FarronF) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.ts:63]: They can be called in but they won't have a button to join.

**Review by FarronF — APPROVED** [review]

**Pull request merged**